client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -43,27 +43,30 @@ const RegisteredDomain = React.createClass( {
 	getPrivacyProtection() {
 		if ( this.props.domain.hasPrivacyProtection ) {
 			return (
-				<Notice
-					isCompact
-					status="is-success"
-					icon="lock">
-					{ this.translate( 'On', {
-						context: 'An icon label when Privacy Protection is enabled.'
-					} ) }
-				</Notice>
+				<a href="#" onClick={ this.goToPrivacyProtection }>
```

Comment:
In that case we should use this path `domainManagementContactsPrivacy` for privacy protection enabled, because at the moment we have redirect in between.